src/collections/components/StoryForm/StoryForm.tsx

```diff
@@ -325,19 +408,35 @@ export const StoryForm: React.FC<StoryFormProps & SharedFormButtonsProps> = (
 
             <Grid item xs={12}>
               <MarkdownPreview minHeight={6.5} source={formik.values.excerpt}>
-                <FormikTextField
-                  id="excerpt"
-                  label="Excerpt"
-                  fieldProps={formik.getFieldProps('excerpt')}
-                  fieldMeta={formik.getFieldMeta('excerpt')}
-                  multiline
-                  minRows={4}
-                />
+                <Grid>
+                  <FormikTextField
+                    id="excerpt"
+                    label="Excerpt"
+                    fieldProps={formik.getFieldProps('excerpt')}
+                    fieldMeta={formik.getFieldMeta('excerpt')}
+                    multiline
+                    minRows={4}
+                  />
+
+                  {descriptionState ==
+                    DescriptionTextStates.CanInsertParserDesc &&
+                    showTextSwitchLink(
+                      'Use Parser Excerpt Instead',
+                      parserItemExcerptText,
+                      DescriptionTextStates.CanRestoreOGDesc
+                    )}
+                  {descriptionState == DescriptionTextStates.CanRestoreOGDesc &&
+                    showTextSwitchLink(
+                      'Restore OpenGraph Description',
+                      ogExcerptText,
+                      DescriptionTextStates.CanInsertParserDesc
+                    )}
+                </Grid>
               </MarkdownPreview>
             </Grid>
             <Grid item xs={12}>
               <Button buttonType="hollow" onClick={fixExcerpt}>
-                Fix excerpt
+                Fix quotes
```

Comment:
Was this intentional? onClick is pointing to the excerpt function 🤔 haven't looked at the UI so I could be wrong here.

Reply:
the code only fixes the quotes so I thought it would be more clear. maybe it is confusing?